examples/cifar10/readme.md

```diff
@@ -31,8 +31,9 @@ nn << conv(32, 32, 5, 3, n_fmaps, padding::same)
     << pool(16, 16, n_fmaps, 2)
     << conv(8, 8, 5, n_fmaps, n_fmaps2, padding::same)
     << pool(8, 8, n_fmaps2, 2)
-    << fully_connected_layer<activation::identity>(4 * 4 * n_fmaps2, n_fc)
-    << fully_connected_layer<softmax>(n_fc, 10);
+    << fully_connected_layer(4 * 4 * n_fmaps2, n_fc)
```

Comment:
renaming to `fully`, `linear` or `dense` will save file space. Applicable for the rest of examples